testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/management/api/SocketsAndInterfacesTestCase.java

```diff
@@ -81,6 +81,7 @@ public void before() throws IOException {
             testNic = getNic(testHost);
         } else {
             testNic = getNonDefaultNic();
+            org.junit.Assume.assumeFalse("No usable non-default nic is available", testNic == null);
```

Comment:
Not that prior to this change testNic being null was also possible and would have produced an NPE. Either way, if a suitable nic cannot be found this test should be skipped.